tests/spec/core/webidl-contiguous-spec.js

```diff
@@ -37,7 +37,13 @@ describe("Core - Contiguous WebIDL", function () {
             expect($target.text()).toEqual(text);
             expect($target.find(".extAttr").text()).toEqual("Something");
             expect($target.find(".idlCtor").text()).toEqual("Constructor()");
-
+            
+            $target = $("#if-identifier-list", doc);
+            text = "[Global=Window,\n Exposed=(Window,Worker)]\ninterface SuperStar {\n};";
```

Comment:
Double-checking that we want the rendered version to be `(X,Y)` instead of `(X, Y)`. (space vs without). http://heycam.github.io/webidl/#Exposed shows it without the space.